src/shared/components/carousel/index.js

```diff
@@ -18,73 +19,49 @@ const settings = {
   nextArrow: <Arrow direction="right"/>
 }
 
-class Carousel extends Component {
-  state = {
-    isMobile: false
-  }
-
-  componentDidMount () {
-    window.addEventListener('resize', this.handleResize)
+const Carousel = ({ itemsList, modifier, size, onVideoClick, activeIndex, translationsList, isMobile }) => {
+  let items
+  const numberOfSlidesToShow = isMobile ? 1 : size
+  const finalSettings = { ...settings, slidesToShow: numberOfSlidesToShow }
 
```

Comment:
This carousel should be context agnostic and not dependent on the usage like it is being used. All in all, should exist a `prop` items instead of the specific logic being here

I know that this was already implemented this way, so please add a `TODO` comment now above the `modifier line`. Ideally, create a non priority Issue and link it as well.

Reply:
I'll open an issue to refactor this component.